docs/config.md

```diff
@@ -168,4 +189,22 @@ spec:
                   name: cryostat-sample-grafana
                   port:
                     number: 3000
+    reportsConfig:
+      annotations:
+        nginx.ingress.kubernetes.io/backend-protocol : HTTPS
+      ingressSpec:
+        tls:
+        - {}
+        rules:
+        - host: testing.cryostat-reports
+          http:
+            paths:
+            - path: /
+              pathType: Prefix
+              backend:
+                service:
+                  name: cryostat-sample-reports
+                  port:
+                    number: 10000
+```
```

Comment:
I think there is no Ingress config for the reports service, so this change can be removed. We don't need to expose the report generator service outside the cluster do we?

Reply:
True, no, I don't think there is really any reason to expose the service outside the cluster. Clients can make requests through the normal Cryostat HTTP API and it'll get internally routed to the sidecar containers.